end_to_end_tests/golden-record/my_test_api_client/api/bodies/post_bodies_multiple.py

```diff
@@ -8,12 +8,19 @@
 from ...models.post_bodies_multiple_data_body import PostBodiesMultipleDataBody
 from ...models.post_bodies_multiple_files_body import PostBodiesMultipleFilesBody
 from ...models.post_bodies_multiple_json_body import PostBodiesMultipleJsonBody
-from ...types import File, Response
+from ...types import UNSET, File, Response, Unset
 
 
 def _get_kwargs(
     *,
-    body: PostBodiesMultipleJsonBody | File | PostBodiesMultipleDataBody | PostBodiesMultipleFilesBody,
+    body: PostBodiesMultipleJsonBody
+    | Unset
+    | File
+    | Unset
+    | PostBodiesMultipleDataBody
+    | Unset
+    | PostBodiesMultipleFilesBody
+    | Unset = UNSET,
```

Comment:
Unions still aren't quite right. This shouldn't list `Unset` twice and in a different, local union test it was trying to call `.to_dict()` on an `Unset`